daemon/mgr/container.go

```diff
@@ -295,8 +295,28 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		HostConfig: config.HostConfig,
 	}
 
+	// store image's config into container's meta
+	imageConfig, err := mgr.Client.GetImageConfig(ctx, config.Image)
+	if err != nil {
+		return nil, err
+	}
+	if len(meta.Config.Entrypoint) == 0 {
```

Comment:
Could you set all these setting codes into a separated function which is name by `func MergeConfig(config1 types.ContainerConfig, config2 v1.ImageConfig) types.ContainerConfig`, or something like this?

Since I think we should encapsulate more in func `Create`. WDYT? @skoo87